static/js/app.js

```diff
@@ -171,6 +179,13 @@ function toggleMenu($menu) {
   }
 }
 
+function expandToggledCategory(expandToggle) {
+  var category = $(expandToggle).attr('class').split(' ')[1]
```

Comment:
This looks quite fragile? is there another way we could do it? perhaps a data attribute?